lib/common/zstd_internal.h

```diff
@@ -186,6 +186,10 @@ typedef enum { set_basic, set_rle, set_compressed, set_repeat } symbolEncodingTy
 #define OffFSELog   8
 #define MaxFSELog  MAX(MAX(MLFSELog, LLFSELog), OffFSELog)
 
+#define ZSTD_MAX_HUF_HEADER_SIZE 128 /* header + <= 127 byte tree description */
+/* Each table cannot take more than #symbols * FSELog bits */
+#define ZSTD_MAX_FSE_HEADERS_SIZE ((MaxML * MLFSELog + MaxLL * LLFSELog + MaxOff * OffFSELog + 7) / 8)
```

Comment:
@Cyan4973 can you double check these numbers? I'm using them to reduce the stack usage of superblock mode, to avoid >1024 byte frame warnings.